sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/data_asset.py

```diff
@@ -0,0 +1,38 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Any, Dict, Optional
+
+from azure.ai.ml._schema._deployment.online.data_asset_schema import DataAssetSchema
+from azure.ai.ml._utils._experimental import experimental
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
+
+
+@experimental
+class DataAsset:
```

Comment:
Do we need a new class, is there any issue with reuse of the existing data version or data container objects?

Reply:
I have a bias to reuse, but if there's good reason it must be different that's ok too.

Reply:
The existing data schema didn't except version or name which is something that will be allowed by the yaml, so we decided it would be best to create a new schema and entity